src/devices/src/virtio/mod.rs

```diff
@@ -60,6 +60,8 @@ pub const VIRTIO_MMIO_INT_CONFIG: u32 = 0x02;
 /// queue events.
 pub const NOTIFY_REG_OFFSET: u32 = 0x50;
 
+pub const QUEUE_SIZE: u16 = 256;
```

Comment:
This change doesn't seem related to the builder tests.

The queue size is also a device specific configuration, so I am not super sure we want to have it globally used even though it's the same value everywhere.

Reply:
I thought because it is the same, we can make it common to the block and net device. However, I understand your concern, and I will get rid of these changes.